sqlglot/expressions.py

```diff
@@ -5501,9 +5501,9 @@ class JSONArrayContains(Binary, Predicate, Func):
 
 class ParseJSON(Func):
     # BigQuery, Snowflake have PARSE_JSON, Presto has JSON_PARSE
+    # Snowflake also has TRY_PARSE_JSON, which is represented using `safe`
     _sql_names = ["PARSE_JSON", "JSON_PARSE"]
-    arg_types = {"this": True, "expressions": False}
-    is_var_len_args = True
```

Comment:
FYI the `is_var_lane_args` attribute seemed unneeded, see relevant [commit](https://github.com/tobymao/sqlglot/commit/d75d760778973d88325e0b6d167243385140d698) that added it